docs/pages/pmd/rules/java/errorprone.md

```diff
@@ -2790,7 +2790,7 @@ Avoid returning from a finally block, this can discard exceptions.
 
 **This rule is defined by the following XPath expression:**
 ``` xpath
-//FinallyStatement//ReturnStatement
+//FinallyStatement//ReturnStatement except //FinallyStatement//(MethodDeclaration|LambdaExpression)//ReturnStatement
```

Comment:
You do not need to add the documentation changes to the commit - these changes will be committed automatically by our CI build. But it doesn't hurt either.

Reply:
Thanks For the Information.